### ISSUE TITLE
Master branch should be renamed to main

### ISSUE THREAD

**Issue by khampf** [khampf/g13#16]
Should rename master branch to main to keep up with times like [Hanselman](https://www.hanselman.com/blog/EasilyRenameYourGitDefaultBranchFromMasterToMain.aspx) suggests

**Comment by khampf** [comment]
This can not be done before I also update the Arch Linux PKGBUILD as it references the "master" branch

**Comment by Techjar** [comment]
No, this is a stupid ideal perpetrated by people who want to censor the internet.

**Comment by khampf** [comment]
I get that it is "master" in the sense of master and copy, not master and slave. However this is the principle for naming the default branch here on github as of late, [Github is clearly moving in that direction](https://www.cnet.com/news/microsofts-github-is-removing-coding-terms-like-master-and-slave/).
As I really do not care what the default branch is named as long it is clear it is "the big one", I see no point in clinging to the name "master" for it either. Roll with the punches

**Comment by Techjar** [comment]
Yeah, only because the people who run GitHub have given in to same nonsense. I'm not just disagreeing with the rename in this repo, I'm disagreeing with the whole movement. It's entirely based on a false premise, and just creates extra work (however little) for developers for no good reason. I could go on and on, and far more in depth into the issue, but I don't want to get too political here.

It's not that I even care what the branch is named, I'm only against renaming it for this particular reason, purely as a matter of principle. There's plenty of projects that don't have a `master` branch at all for other reasons (e.g. Minecraft mods that have a branch for each game version), and those are perfectly valid.
